pkg/cloudprovider/drift.go

```diff
@@ -31,10 +31,10 @@ import (
 )
 
 const (
-	AMIDrift                cloudprovider.DriftReason = "AMIDrift"
-	SubnetDrift             cloudprovider.DriftReason = "SubnetDrift"
-	SecurityGroupDrift      cloudprovider.DriftReason = "SecurityGroupDrift"
-	NodeTemplateStaticDrift cloudprovider.DriftReason = "NodeTemplateStaticDrift"
+	AMIDrift           cloudprovider.DriftReason = "AMIDrift"
+	SubnetDrift        cloudprovider.DriftReason = "SubnetDrift"
+	SecurityGroupDrift cloudprovider.DriftReason = "SecurityGroupDrift"
+	NodeTemplateDrift  cloudprovider.DriftReason = "NodeTemplateDrift"
```

Comment:
We have `ProvisionerStaticallyDrifted` in one place and just `NodeTemplateDrift` here. We should be consistent. If we are going to do `ProvisionerStaticallyDrifted` we should do `NodeTemplateStaticallyDrifted`

Reply:
Updated Here: https://github.com/aws/karpenter-core/pull/474